header_checks/meson.build

```diff
@@ -221,6 +221,11 @@ endif
 # libcheck
 check = dependency('check', fallback : ['check', 'check_dep'])
 
+if sys_windows
+  lua_project = subproject('lua')
+  lua = lua_project.get_variable('lua_dep')
+endif
+
```

Comment:
This is not necessary anymore (due to that change with the `fallback` right there below.